README.md

```diff
@@ -248,27 +248,23 @@ export NVM_DIR="$HOME/.nvm"
 
 ### Multiple commands
 
-By design and just like `scripts` defined in `package.json`, husky will run hook scripts as a single command. 
+To run multiple commands in series you can specify an array of commands.
 
 ```json
-"pre-commit": "cmd && cmd"
+"pre-commit": [
+  "cmd",
+  "cmd"
+]
 ```
 
-That said, if you prefer to use an array, the recommended approach is to define them in `.huskyrc.js` or `husky.config.js`.
+Or if you use a string instead, just like `scripts` defined in `package.json`, husky will run hook scripts as a single command. 
 
-```js
-const tasks = arr => arr.join(' && ')
-
-module.exports = {
-  'hooks': {
-    'pre-commit': tasks([
-      'cmd',
-      'cmd'
-    ])
-  }
-}
+```json
+"pre-commit": "cmd1 && cmd2"
```

Comment:
Now one of the example will execute "cmd1 && cmd2" and the other "cmd && cmd" which is not consistent :confused: Would you mind updating this as below (or the other snippet to use "cmd1" and "cmd2")?

```suggestion
"pre-commit": "cmd && cmd"
```